features/upgrade/samples_operator/upgrade.feature

```diff
@@ -0,0 +1,47 @@
+    Given the expression should be true> cluster_operator('openshift-samples').condition(type: 'Degraded')['status'] == "False"
+    # Check cluster operator image-registry should be in correct status
+    Given the expression should be true> cluster_operator('image-registry').condition(type: 'Available')['status'] == "True"
+    Given the expression should be true> cluster_operator('image-registry').condition(type: 'Progressing')['status'] == "False"
+    Given the expression should be true> cluster_operator('image-registry').condition(type: 'Degraded')['status'] == "False"
+    # Check cluster operator openshift-controller-manager should be in correct status
+    Given the expression should be true> cluster_operator('openshift-controller-manager').condition(type: 'Available')['status'] == "True"
+    Given the expression should be true> cluster_operator('openshift-controller-manager').condition(type: 'Progressing')['status'] == "False"
+    Given the expression should be true> cluster_operator('openshift-controller-manager').condition(type: 'Degraded')['status'] == "False"
+
+  # @author xiuwang@redhat.com
+  @upgrade-check
+  @users=upuser1,upuser2
+  @admin
+  Scenario: samples/openshift-controller-manager/imare-registry operators should be in correct status after upgrade
+    Given I switch to cluster admin pseudo user
+    # Check cluster operator openshift-samples should be in correct status
+    Given the expression should be true> cluster_operator('openshift-samples').condition(type: 'Available')['status'] == "True"
+    Given the expression should be true> cluster_operator('openshift-samples').condition(type: 'Progressing')['status'] == "False"
+    Given the expression should be true> cluster_operator('openshift-samples').condition(type: 'Degraded')['status'] == "False"
+    # Check cluster operator image-registry should be in correct status
+    Given the expression should be true> cluster_operator('image-registry').condition(type: 'Available')['status'] == "True"
+    Given the expression should be true> cluster_operator('image-registry').condition(type: 'Progressing')['status'] == "False"
+    Given the expression should be true> cluster_operator('image-registry').condition(type: 'Degraded')['status'] == "False"
+    # Check cluster operator openshift-controller-manager should be in correct status
+    Given the expression should be true> cluster_operator('openshift-controller-manager').condition(type: 'Available')['status'] == "True"
+    Given the expression should be true> cluster_operator('openshift-controller-manager').condition(type: 'Progressing')['status'] == "False"
+    Given the expression should be true> cluster_operator('openshift-controller-manager').condition(type: 'Degraded')['status'] == "False"
+
+    Given I switch to the first user
+    Given I have a project
+    When I run the :new_app client command with:
```

Comment:
I suggest not run this step, since it will be failed in disconencted cluster. Let's just keep this to check operator status is well before/after upgrade.